kernels/gemmini/Snakefile

```diff
@@ -0,0 +1,86 @@
+from util.snake.paths import get_default_paths
+from util.snake.flags import get_mlir_preproc_flags
+
+
+config.update(get_default_paths())
+config["snaxoptflags"] = ",".join(
+    [
+        "insert-accfg-op{accelerator=gemmini}",
+        "convert-linalg-to-accfg",
+        "convert-accfg-to-csr",
+    ]
+)
+
+# Rocket-specific RV flags
+config["cflags"] = [
+    "-c",
+    "-O3",
+    "-Wno-override-module",
+    "--target=riscv64-unknown-elf",
+    "-mcpu=generic-rv64",
+    "-march=rv64gc",
+]
+
+config["mliroptflags"] = [
+    "--pass-pipeline='builtin.module(transform-interpreter{debug-bind-trailing-args=linalg.quantized_matmul}, test-transform-dialect-erase-schedule, linalg-generalize-named-ops)'"
+]
+config["mlirpreprocflags"] = get_mlir_preproc_flags()
+
+
+module default_rules:
+    snakefile:
+        "../../util/snake/default_rules.smk"
+    config:
+        config
+
+
+use rule preprocess_mlir from default_rules as default_preprocess_mlir
+
+
+use rule translate_mlir from default_rules as default_translate_mlir
```

Comment:
why the renaming?

Reply:
This is to indicate that we import them from the defaults.
```
use rule preprocess_mlir, translate_mlir from default_rules as default_*
```
Also works, but snakefmt wouldn't let me, maybe the newer versions fix this?